protocol/blockchain/script.md

```diff
@@ -184,6 +184,25 @@ Numeric opcodes (OP_1ADD, etc) are restricted to operating on 4-byte integers. T
 | OP_CHECKSEQUENCEVERIFY | 178     | 0xb2      | x     |x / *fail* |  Marks transaction as invalid if the relative lock time of the input (enforced by BIP68 with nSequence) is not equal to or longer than the value of the top stack item. The precise semantics are described in [BIP112](/protocol/forks/bip-0112). |
 
+### Introspection
+
+|Word                    |Value | Hex  |Input           |Output   | Description                                            |
+|------------------------|------|------|----------------|---------|--------------------------------------------------------|
+| OP_INPUTINDEX          | 192  | 0xc0 | Nothing        | number  | Push the index of the input being evaluated to the stack as a Script Number.   |
+| OP_ACTIVEBYTECODE      | 193  | 0xc1 | Nothing        | script  | Push the bytecode currently being evaluated, beginning after the last executed OP_CODESEPARATOR, to the stack1. For Pay-to-Script-Hash (P2SH) evaluations, this is the redeem bytecode of the Unspent Transaction Output (UTXO) being spent; for all other evaluations, this is the locking bytecode of the UTXO being spent.  |
+| OP_TXVERSION           | 194  | 0xc2 | Nothing        | number  | Push the version of the current transaction to the stack as a Script Number. |
+| OP_TXINPUTCOUNT        | 195  | 0xc3 | Nothing        | number  | Push the count of inputs in the current transaction to the stack as a Script Number. |
+| OP_TXOUTPUTCOUNT       | 196  | 0xc4 | Nothing        | number  | Push the count of outputs in the current transaction to the stack as a Script Number.  |
+| OP_TXLOCKTIME          | 197  | 0xc5 | Nothing        | number  | Push the locktime of the current transaction to the stack as a Script Number. |
+| OP_UTXOVALUE           | 198  | 0xc6 | index          | number  | Pop the top item from the stack as an input index (Script Number). Push the value (in satoshis) of the Unspent Transaction Output (UTXO) spent by that input to the stack as a Script Number. |
+| OP_UTXOBYTECODE        | 199  | 0xc7 | index          | script  | Pop the top item from the stack as an input index (Script Number). Push the full locking bytecode of the Unspent Transaction Output (UTXO) spent by that input to the stack. |
+| OP_OUTPOINTTXHASH      | 200  | 0xc8 | index          | hash    | Pop the top item from the stack as an input index (Script Number). From that input, push the outpoint transaction hash � the hash of the transaction which created the Unspent Transaction Output (UTXO) which is being spent � to the stack in OP_HASH256 byte order. |
```

Comment:
Some illegal character seems to have sneaked into this line and the next as well (4 total)
git diff shows them as `<96>`.